cloudformation/batch-substack.cfn.json

```diff
@@ -145,6 +145,25 @@
                 }
               ]
             }
+          },
+          {
+            "PolicyName": "cfnDescribeStacks",
+            "PolicyDocument": {
+              "Version": "2012-10-17",
+              "Statement": [
+                {
+                  "Action": [
+                    "cloudformation:DescribeStacks"
+                  ],
+                  "Effect": "Allow",
+                  "Resource": [
+                    {
+                      "Fn::Sub": "arn:${AWS::Partition}:cloudformation:${AWS::Region}:${AWS::AccountId}:stack/${ClusterName}*"
```

Comment:
why the final '*' ?

Reply:
Without the `*` I have this error:
```
An error occurred (AccessDenied) when calling the DescribeStacks operation: 
User: arn:aws:sts::<accountid>:assumed-role/parallelcluster-<name>-AWSBatchStack-CCW1ZPE-JobRole-XK7BOLJ50HZR/2ea928bf-31ab-4672-aa9b-9af415480909 
is not authorized to perform: cloudformation:DescribeStacks on resource: 
arn:aws:cloudformation:eu-west-1:499649255560:stack/parallelcluster-<name>/*
```

I changed it to: `...${ClusterName}/*`